### PR TITLE
INFRA-2261: add DGB experimental v3 using staging endpoint

### PR DESCRIPTION
**Pull request by gre**
PR replaces #914 with latest iteration.
it is not meant to be merged but to run on CI to trigger the tests and/or be reused to try txs with.

it works with EXPERIMENTAL_EXPLORERS env on

### CODE REVIEW

**Comment by lgillot-lamure-ledger** [comment]
:+1:

**Comment by lgillot-lamure-ledger** [comment]
BTW you can add LTC too !

File: src/api/explorerConfig.js [hunk 1]
```diff
@@ -71,6 +71,10 @@ export const explorerConfig: { [id: CryptoCurrencyIds]: ?Config } = {
       base: "EXPLORER",
       version: "v2",
     },
+    experimental: {
+      base: "EXPLORER_BETA",
+      version: "v3"
```
Comment (lgillot-lamure-ledger) [inline src/api/explorerConfig.js:76]:
```suggestion
      version: "v3",
```

**Comment by gre** [comment]
continues on https://github.com/LedgerHQ/ledger-live-common/pull/923

**Pull request closed without merging**